src/delphi/train/tokenized_chunks_dataset.py

```diff
@@ -0,0 +1,57 @@
+import torch
+from torch.utils.data import Dataset
+
+from delphi.train.shuffle import shuffle_list
+
+
+class TokenizedChunksDataset(Dataset):
+    def __init__(self, tokenized_docs, max_seq_len, device):
+        self.device = device
+        self.tokenized_docs = tokenized_docs
+        self.doc_len = len(tokenized_docs[0]["tokens"])
```

Comment:
"doc_len" = "document_length" = context length?

Reply:
Document length, but I don't think that's context length. `max_seq_len` is model context length - or more accurately, how long training samples are.